assets/src/domain/eventEditor/services/apollo/queries/datetimes/useDatetimes.ts

```diff
@@ -1,12 +1,19 @@
+import { useMemo } from 'react';
+
 import useDatetimeQueryOptions from './useDatetimeQueryOptions';
 import { Datetime, DatetimesList } from '../../types';
 import useCacheQuery from '../useCacheQuery';
+import { getCacheIds } from '@sharedServices/predicates';
 
 const useDatetimes = (): Array<Datetime> => {
 	const options = useDatetimeQueryOptions();
 	const { data } = useCacheQuery<DatetimesList>(options);
 
-	return data?.espressoDatetimes?.nodes || [];
+	const nodes = data?.espressoDatetimes?.nodes || [];
+
+	const cacheIds = getCacheIds(nodes).join(':');
```

Comment:
I noticed here: https://github.com/eventespresso/event-espresso-core/pull/2630/files#diff-8732518b6f67c185fb6198f60a1c3647R156

that you were using `:` in the cache IDs generated server side. 

Will that conflict with using the same character here to separate IDs?

maybe we should use `-` or `,` here ?

Reply:
It doesn't matter at all. Here is an array of `cacheId`s of the whole list that I convert to string. `.join(':')` is simpler than `JSON.stringify()`